src/Adapter/Product/QueryHandler/GetProductForEditingHandler.php

```diff
@@ -63,21 +57,13 @@ class GetProductForEditingHandler extends AbstractProductHandler implements GetP
      */
     private $numberExtractor;
 
-    /**
-     * @var GetProductSuppliersForEditingHandlerInterface
-     */
-    private $getProductSuppliersForEditingHandler;
-
     /**
      * @param NumberExtractor $numberExtractor
-     * @param GetProductSuppliersForEditingHandlerInterface $getProductSuppliersForEditingHandler
      */
     public function __construct(
-        NumberExtractor $numberExtractor,
-        GetProductSuppliersForEditingHandlerInterface $getProductSuppliersForEditingHandler
+        NumberExtractor $numberExtractor
```

Comment:
Please don't remove the second argument, it's a BC.

Reply:
@Progi1984 It's not a BC because this class was not released yet 😉  . Until this code is part of a release we can modify it at will.

Reply:
Ok. Correct in this case.